core/css/decanter.css

```diff
@@ -6496,9 +6496,14 @@ dfn {
 
 .su-alert--info {
   background-color: #00548f;
-  background-image: url("../img/alert-info.png");
+  background: url("../img/alert-info.png");
```

Comment:
Making this `background:` instead of `background-image:` overrides the `background-color:`. Therefore the blue color is now appearing. Also, the image is repeating if we don't tell it not to.

Reply:
@josephgknox Good catch! I just pulled down the branch and was wondering why everything was white 😺